### ISSUE TITLE
User Management - UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#20]
## User story
1. As a user (admin)
2. I want / need to be able to see all registered users and registration requests
3. So that I can add new users or give certain users admin priviledges or other permissions

## Acceptance criteria
* The user management page should be identifyable as such
* The user management page should only be accessible for admins
* The user management page should provide a tab with an overview of all registered users with their data and status in a list and a tab showing all registration requests of new users with their provided data in a list
* The page should provide a search bar to enable the admin to quickly find users
* The tab showing all registered users should provide a button to delete a user and edit a user's permissions
* The tab showing the registration requests should provide a button for approving and declining the registration

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
